aptos-move/framework/move-stdlib/sources/cmp.move

```diff
@@ -0,0 +1,98 @@
+module std::cmp {
+    const EQUAL: u8 = 1;
+    const LESS_THAN: u8 = 0;
+    const GREATER_THAN: u8 = 2;
+
+    enum Ordering has copy, drop {
+        /// First value is less than the second value.
+        LessThan,
+        /// First value is equal to the second value.
+        Equal,
+        /// First value is greater than the second value.
+        GreaterThan,
+    }
+
+    public native fun compare<T>(first: &T, second: &T): Ordering;
```

Comment:
Please add a `///` doc comment to this function. You should likely outline that you do lexicographical ordering, with one or two examples.